package.json

```diff
@@ -65,6 +65,11 @@
     "test:one": "polkadot-dev-run-test --env browser",
     "test:skipped": "echo 'tests skipped'"
   },
+  "dependencies": {
+    "@azns/resolver-core": "^1.4.0",
+    "@azns/resolver-react": "^1.5.0",
+    "loglevel": "^1.8.1"
```

Comment:
Is this used anywhere?

Reply:
very funny story :angry: 
it's a peerDependency of @azns/resolver-core or @azns/resolver-react, but apparently yarn doesn't install peerDependencies if they are missing and all it give you is a warning. This solves build failing